src/bitcoin-cli.cpp

```diff
@@ -108,7 +108,7 @@ static int AppInitRPC(int argc, char* argv[])
         if (!gArgs.IsArgSet("-version")) {
             strUsage += "\nUsage:\n"
                   "  bitcoin-cli [options] <command> [params]  " + strprintf("Send command to %s", PACKAGE_NAME) + "\n" +
-                  "  bitcoin-cli [options] -named <command> [name=value] ... " + strprintf("Send command to %s (with named arguments)", PACKAGE_NAME) + "\n" +
+                  "  bitcoin-cli [options] -named <command> [name=value]...  " + strprintf("Send command to %s (with named arguments)", PACKAGE_NAME) + "\n" +
```

Comment:
I think the solution in #13872 is better for help2man.

Reply:
#13872 introduces SYNOPSIS section to the all bitcoind, bitcoin-qt and utilities manual pages. I think that is a good move. And, as a side effect, #13872 fixes one of two bugs in bitcoin-cli --help output.

#13879 just fixes all found bugs in bitcoin-cli --help output and is safe for the whole code.